src/Guardar.js

```diff
@@ -111,6 +111,15 @@ export function has(key) { return keys().includes(key); }
  */
 export function isEmpty() { return size() === 0; }
 
+/**
+ * Get version number
+ * 
+ * @returns {number} number of current version 
+ */
+export function getVersion() {
+	return("1.0.5");
```

Comment:
I would say to remove the parentheses, but it's not a big deal, and I'd be nitpicking

Reply:
Ah its fine its good practice, I made the changes.